contrib/ipfs-usb/movestore

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+if [ -z "$1" ] && [ -z "$2" ]; then
+   echo Syntax 
+   echo movestore \[Device\] \[User\]
+   echo Device - Device to format and mount
+   echo User - User running ipfs that the repo will be moved to
+   exit 0
+fi
+device="$1"
+user="$2"
+
+mkfs.ext3 /dev/$device
+service ipfs stop
+e2label  /dev/$device ipfs-data
+mkdir /mnt/ipfs-data
+echo "LABEL=ipfs-data  /mnt/ipfs-data ext3  defaults,noatime  0  0" >> /etc/fstab
+mount /mnt/ipfs-data
+mv /home/$user/.ipfs /mnt/ipfs-data/
+ln -s /mnt/ipfs-data/.ipfs /home/$user/.ipfs
```

Comment:
It might be better to just change the IPFS_PATH var, that way you don't have to remove symlinks and stuff later. Also it might be good to have a script to undo this.

Reply:
I'd like to revisit the IPFS_PATH in favor of a user instead anyway.